core/src/components/app/app.tsx

```diff
@@ -17,18 +17,17 @@ export class App implements ComponentInterface {
   @Element() el!: HTMLElement;
 
   /**
-   * Used to set focus on an element that uses `ion-focusable`.
-   * Do not use this if focusing the element as a result of a keyboard
-   * event as the focus utility should handle this for us. This method
-   * should be used when we want to programmatically focus an element as
-   * a result of another user action. (Ex: We focus the first element
-   * inside of a popover when the user presents it, but the popover is not always
-   * presented as a result of keyboard action.)
```

Comment:
Why did this comment get removed here? This is what ends up in the documentation here: https://ionicframework.com/docs/api/app#setfocus

Reply:
It made sense to me to remove it since `focusElements` has it but I'm not committed to it. I added it back: https://github.com/ionic-team/ionic-framework/pull/30218/commits/2edbdaffdcc5f80c162a52b0687a2861acf9ba6c